pkg/auth/rbac.go

```diff
@@ -185,6 +191,43 @@ func ValidateRbacPermission(line string) (p Permission, err error) {
 	}, nil
 }
 
+func ValidateTeamRbacPermission(line string, teamPermissions *RBACTeams) (team string, err error) {
```

Comment:
The function name surprises me. Given that it starts with "Validate" I would expect that it just validates. But it also changes the `teamPermission` parameter.
I'd prefer if the actuall adding of values to `teamPermission` is done outside this function.